core/types/transaction_meta_test.go

```diff
@@ -52,6 +69,8 @@ var (
 func TestTransactionMetaEncode(t *testing.T) {
 	for _, test := range txMetaSerializationTests {
 		txmeta := NewTransactionMeta(test.txid, test.msgSender, test.sighashType)
+		txmeta.QueueOrigin = test.queueOrigin
```

Comment:
This is to prevent the need to change the function signature of `NewTransactionMeta` for now.